s4e-backend/src/main/java/pl/cyfronet/s4e/config/SecurityConfig.java

```diff
@@ -133,7 +133,9 @@ private void authorizeRequests(ExpressionUrlAuthorizationConfigurer<HttpSecurity
                     .access("isAuthenticated() && @licensePermissionEvaluator.allowSceneRead(#id, principal)")
 
                 .mvcMatchers(GET, prefix("/search")).permitAll()
+                .mvcMatchers(GET, prefix("/count")).permitAll()
                 .mvcMatchers(GET, prefix("/dhus/search")).authenticated()
+                .mvcMatchers(GET, prefix("/dhus/count")).authenticated()
```

Comment:
I'd prefix both count endpoints with `/search`, i.e. `/search/count` and `/dhus/search/count` as currently they float without context.